torch/_C/__init__.pyi.in

```diff
@@ -81,7 +80,30 @@ class finfo:
     @overload
     def __init__(self) -> None: ...
 
-${dtype_class_hints}
+float32: dtype = ...
```

Comment:
this is generated. you dont need to explicitly set these.

Reply:
How about now?